ci/Jenkinsfile.groovy

```diff
@@ -127,7 +127,7 @@ ansiColor('xterm') {
                         }
 
                         isMasterJob = (env.CHANGE_BRANCH == null || env.CHANGE_BRANCH == '')
-                        isRelease   = env.CHANGE_BRANCH.startsWith("rel-")
+                        isRelease   = !isMasterJob && env.CHANGE_BRANCH.startsWith("rel-")
```

Comment:
Does adding `!isMasterJob` in this condition really changes anything? If `env.CHANGE_BRANCH.startsWith("rel-") == true` then `!isMasterJob` is automatically `true`...

Reply:
but if `CHANGE_BRANCH` is `null`, then `.startsWith()` will throw an error, which is really bad.

Reply:
Yeah, if Jenkins throws exceptions, that's not good :)